std/algorithm/searching.d

```diff
@@ -2279,7 +2279,7 @@ private R1 simpleMindedFind(alias pred, R1, R2)(R1 haystack, scope R2 needle)
 }
 
 /**
-Finds two or more $(D needles) into a $(D haystack). The predicate $(D
+Finds two or more `needles` into a `haystack`. The predicate $(D
```

Comment:
How about moving the changes of `$(D...)` to backticks to a different PR that we can merge immediately?

Reply:
Let's better follow-up on the various requests to use backticks in the docs and do it for `std.algorithm` once and for all?
-> #5970